app/static/src/app/store/run/state.ts

```diff
@@ -15,6 +16,22 @@ export interface ParameterSet {
     hidden: boolean
 }
 
+type AdConfStandardForm = {
+    val: [number|null, number|null],
```

Comment:
Personally I'd prefer the more verbose alternatives, since I find the mental effort of remembering what an 'AdConf' is greater than reading a longer name.. but that's a very subjective thing, so it's up to you!
```suggestion
type AdvancedConfigStandardForm = {
    val: [number|null, number|null],
```

Reply:
nah i think its easy enough to change and fair enough, i will probably forget what AdConf stands for next year haha

Reply:
done